exporter/exporterparser/zipkin_test.go

```diff
@@ -49,12 +163,19 @@ func TestZipkinExportersFromYAML_roundtripJSON(t *testing.T) {
 	}))
 	defer cst.Close()
 
+	var spans []interface{}
```

Comment:
It would be nice if these could be unmarshaled to a concrete type to help us catch errors/regressions.